server/routes/reporting/standard-report/standard-report.controller.js

```diff
@@ -327,11 +327,22 @@
           }
 
           if (header.id === 'attendance_audit') {
-            // ADD IN POOL ATTENDANCE AUDIT REPORT ROUTE ONCE AVAILABLE
+            let link;
+            if (output.substring(0,1) === 'P') {
+              link = app.namedRoutes.build('reports.pool-attendance-audit.report.get', {
+                filter: output,
+              })
+            } else if (output.substring(0,1) === 'J') {
+              link = app.namedRoutes.build('reports.jury-attendance-audit.report.get', {
+                filter: output,
+              })
+            }
```

Comment:
can we do `output.charAt(0)` here instead?

Reply:
We can indeed, ive changed.

Reply:
We can indeed, ive changed.